docs/source-app/quickstart.rst

```diff
@@ -116,13 +114,6 @@ Simply add ``--cloud`` to run this application in the cloud 🤯
 
     lightning run app app.py --cloud
 
-And with just one line of code, run on cloud GPUs!
-
-.. code-block:: bash
-
-    USE_GPU=1
-    lightning run app app.py --cloud
```

Comment:
This does not work. The quick start code does not have any code that handles this.

Reply:
Yes, this is out-dated. It needs to be passed with --app_args